src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -156,26 +156,26 @@ export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, Art
             {artwork.edition_of}
           </Serif>
         )}
-        {artwork.attribution_class && (
+        {!!artwork.attribution_class && (
           <Serif color="black60" size="3t" mt={1}>
             <TouchableWithoutFeedback onPress={() => this.handleTap("/artwork-classifications")}>
               <Text style={{ textDecorationLine: "underline" }}>{artwork.attribution_class.shortDescription}</Text>
             </TouchableWithoutFeedback>
             .
           </Serif>
         )}
-        {artwork.isInAuction &&
-          artwork.sale &&
+        {!!artwork.isInAuction &&
+          !!artwork.sale &&
           !artwork.sale.isClosed && (
             <>
               <Spacer mb={1} />
-              {artwork.partner && (
+              {!!artwork.partner && (
                 <Serif color="black100" size="3t" weight="semibold">
                   {artwork.partner.name}
                 </Serif>
               )}
-              {artwork.saleArtwork &&
-                artwork.saleArtwork.estimate && (
+              {!!artwork.saleArtwork &&
+                !!artwork.saleArtwork.estimate && (
```

Comment:
What's up with all these double-bangs? I think the only time that casting to boolean actually matters is when the value might be `0`.

Reply:
It's likely to avoid returning an empty string, which is illegal in React Native _outside_ of `<Text>…</Text>`.

Reply:
(This is probably going over-board a bit and I think that might not be the best precedent to set, but for now it's also not to the detriment of its functioning.)